templates/components/llamaindex/typescript/streaming/annotations.ts

```diff
@@ -3,17 +3,17 @@ import { MessageContent, MessageContentDetail } from "llamaindex";
 
 export type DocumentFileType = "csv" | "pdf" | "txt" | "docx";
 
-export type DocumentFileContent = {
-  type: "ref" | "text";
-  value: string[] | string;
+export type UploadedFileMeta = {
+  id: string;
+  name: string;
+  url?: string;
+  refs?: string[];
 };
 
 export type DocumentFile = {
-  id: string;
-  filename: string;
-  filesize: number;
-  filetype: DocumentFileType;
-  content: DocumentFileContent;
+  type: DocumentFileType;
+  url: string;
+  metadata: UploadedFileMeta;
```

Comment:
e.g. here the metadata is always defined

Reply:
This is backend model, which the metadata for the uploaded should be defined.